projects/plugins/jetpack/modules/widgets/top-posts.php

```diff
@@ -454,15 +458,17 @@ public function widget( $args, $instance ) {
 						 */
 						$filtered_permalink = apply_filters( 'jetpack_top_posts_widget_permalink', $post['permalink'], $post );
 
-						printf(
-							'<a href="%1$s" title="%2$s" class="bump-view" data-bump-view="tp"%3$s><img width="%4$d" height="%5$d" src="%6$s" alt="%2$s" data-pin-nopin="true"/></a>',
-							esc_url( $filtered_permalink ),
-							esc_attr( wp_kses( $post['title'], array() ) ),
-							( get_queried_object_id() === $post['post_id'] ? ' aria-current="page"' : '' ),
-							absint( $width ),
-							absint( $height ),
-							esc_url( $post['image'] )
-						);
+						if ( ! empty( $image ) ) {
```

Comment:
This change doesn't really work. You're testing whatever `$image` was left at at the end of the loop, not the image for this particular post.

Here you'll want to do something like
```suggestion
						if ( isset( $post['image'] ) ) {
```
to check that the image got set on this post.

Reply:
Fixed variable reference.